foods.html

```diff
@@ -43,109 +43,8 @@
     </div>
 </div>
 
-  <script type="text/javascript">
```

Comment:
Much better to split this out into a .js file, but I'm not sure if it belongs in a js folder within lib.